cypress/objects/commonItems.js

```diff
@@ -0,0 +1,15 @@
+class commonItems {
+
+    visit() {
+        cy.visit('/tasks')
+    }
+
+    getTasksTab() {
+        cy.get('[href="/tasks"]')
```

Comment:
The use of "return" before cy.** is required to get that object after finding it.
`return cy.get('[href="/tasks"]')`